### ISSUE TITLE
Better Presets Handling

### ISSUE THREAD

**Issue by Sasha-Radojevic** [JeffMcClintock/SynthEdit-Tracking#64]
Hi Jeff,

Top request from my synth plugin users is better preset handling.

Instead of a single folder jammed with presets, it would be cool to have:

1. First level of folders that represent banks.
2. Second level of folders that represent categories.
3. Third level (and all subfolders) filled with presets.

Thanks!

**Comment by Pettson77** [comment]
I really need that too!

Or if you could load vst3 preset files as a file. Then you can use a Lstbox and a file browser with sub paths to load all the presets.

**Comment by Steve4380** [comment]
Yes this is an area that really could do with improvement. This gets my vote.

**Comment by Inkogniflow** [comment]
Way to go!
